packages/presentation/lib/screen/home/home_state.dart

```diff
@@ -0,0 +1,27 @@
+enum GameState { inGame, lose, win }
+
+class HomeState {
+  HomeState({
+    this.generatedNumber,
+    required this.attempts,
+    required this.isSubmitButtonActive,
+    required this.gameState,
+  });
+
+  int? generatedNumber;
```

Comment:
I prefer to use final fields. In bloc you just create new state class with new data (or copyWith method) and emit it. but up to you

Reply:
I prefer my way

Reply:
agree with @vladimir-hodaryonok-idf. if it's possible to make smth final - make it

Reply:
agree, but not in this case, prefer use mutable variable, because their easier to send + we use method copy in bloc base

Reply:
corrected